apps/backend/src/modules/auth/auth.controller.ts

```diff
@@ -72,8 +72,13 @@ class AuthController extends BaseController {
 			body: UserSignUpRequestDto;
 		}>,
 	): Promise<APIHandlerResponse> {
+		const { token, user } = await this.authService.signUp(options.body);
+
 		return {
-			payload: await this.authService.signUp(options.body),
+			payload: {
+				token,
```

Comment:
Save this token in frontend, since we already have signUp action there. Use storage module that is already created, do not create a new helper